src/preferences/pages/general.ts

```diff
@@ -23,6 +23,7 @@ export const General = GObject.registerClass(
             'skip_libhandy',
             'border_width',
             'border_color',
+            'unfocused_border_color',
```

Comment:
If you're calling this `unfocused_border_color` it probably makes sense to rename the other one to `focused_border_color`